doc/source/installation.rst

```diff
@@ -44,9 +44,26 @@ In |Percona Server| :rn:`5.6.24-72.2` and newer, the single binary tarball was r
  * ssl101 - for *CentOS* 6 and *CentOS* 7 (``libssl.so.10 => /usr/lib64/libssl.so.10 (0x00007facbe8c4000)``);
  * ssl098e - to be used only for *CentOS* 5 (``libssl.so.6 => /lib64/libssl.so.6 (0x00002aed5b64d000)``).
 
-You can download the binary tarballs from the ``Linux - Generic`` `section <https://www.percona.com/downloads/Percona-Server-5.6/LATEST/binary/tarball/>`_ on the download page.
+.. note::
+
+        The list is a guide. To find which libssl.so files are available on your
+        system you should run the following in Ubuntu or Debian:
+
+        .. code-block:: bash
+
+            $ locate libssl | grep /usr/lib/
```

Comment:
I would better replace this with: `locate libssl | grep "^/usr/lib/"`
since the command you listed lists the libssl libraries from the installed snaps also, which probably should not be used for this.
Here's how it looks on ubuntu bionic:
```
$ locate libssl | grep /usr/lib/
/snap/core18/1650/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/snap/core18/1650/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/snap/core18/1668/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/snap/core18/1668/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/snap/gnome-3-28-1804/110/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/snap/gnome-3-28-1804/116/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/usr/lib/firefox/libssl3.so
/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/usr/lib/x86_64-linux-gnu/libssl3.so
```
this one gives shorter list without the ones from the snaps (might be less confusing for users).
```
$ locate libssl | grep "^/usr/lib/"
/usr/lib/firefox/libssl3.so
/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/usr/lib/x86_64-linux-gnu/libssl3.so
```

Reply:
Done. I changed the command to match your suggestion.